proto/orc_proto.proto

```diff
@@ -74,6 +74,13 @@ message BinaryStatistics {
   optional sint64 sum = 1;
 }
 
+// ORC-398: Statistics for list and map
```

Comment:
I don't think annotating the jira number helps here.

Reply:
Fixed it.